pkg/controller/pcideviceclaim/pcideviceclaim_controller.go

```diff
@@ -100,6 +148,13 @@ func unbindPCIDeviceFromVfioPCIDriver(addr string) error {
 	return nil
 }
 
+func addHostDeviceToKubeVirtAllowList(pd *v1beta1.PCIDevice) {
+	// Get KubeVirt CR
+	// Get PermittedHostDevices
+	// If pd is not in PermittedHostDevices,
+	// then add it and update the CR
+}
+
```

Comment:
This is empty, should be deleted

Reply:
On second thought, this should exist, since there are scenarios where I need to enable allow-listing without doing the vfio-pci driver binding. Making them separate functions is better